src/bindings/interface/analysermodel.i

```diff
@@ -0,0 +1,128 @@
+%module(package="libcellml") analysermodel
+
+#define LIBCELLML_EXPORT
+
+%include <std_shared_ptr.i>
+
+%feature("docstring") libcellml::AnalyserModel
+"Creates an :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::isValid
+"Tests if this :class:`AnalyserModel` object is valid.";
+
+%feature("docstring") libcellml::AnalyserModel::type
+"Returns the :enum:`AnalyserModel::Type`.";
+
+%feature("docstring") libcellml::AnalyserModel::voi
+"Returns the :class:`AnalyserVariable` for the variable of integration.";
+
+%feature("docstring") libcellml::AnalyserModel::voi
+"Returns the :class:`AnalyserVariable` for the variable of integration.";
+
+%feature("docstring") libcellml::AnalyserModel::stateCount
+"Returns the number of states contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::states
+"Returns the states contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::state
+"Removes the state, specified by index, contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::variableCount
+"Returns the number of variables contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::variables
+"Returns the variables contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::variable
+"Removes the variable, specified by index, contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::equationCount
+"Returns the number of equations contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::equations
+"Returns the equations contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::equation
+"Removes the equation, specified by index, contained by this :class:`AnalyserModel` object.";
+
+%feature("docstring") libcellml::AnalyserModel::needEqFunction
+"Tests if this :class:`AnalyserModel` object needs the equality function.";
+
+%feature("docstring") libcellml::AnalyserModel::needNeqFunction
+"Tests if this :class:`AnalyserModel` object needs the inequality function.";
+
+%feature("docstring") libcellml::AnalyserModel::needLtFunction
+"Tests if this :class:`AnalyserModel` object needs the lower than function.";
+
+%feature("docstring") libcellml::AnalyserModel::needLeqFunction
+"Tests if this :class:`AnalyserModel` object needs the lower or equal than function.";
```

Comment:
less than or equal to

Reply:
Should be fixed as part of commit 4762f45.